ruby/anagram/anagram.rb

```diff
@@ -0,0 +1,26 @@
+class Anagram
+
+  def initialize(word)
+    @word = word
+  end
+
+  def match(potential_angrms)
+    anagram_list = []
+    remove_identical_words!(potential_angrms)
+    potential_angrms.each_with_index do |word, index|
+      if canonical(word) == canonical(@word)
+        anagram_list << potential_angrms[index]
```

Comment:
Is the array lookup necessary since you have access to the `word` variable from line 10?